odoo/models.py

```diff
@@ -4338,15 +4338,7 @@ def exists(self):
         query = """SELECT id FROM "%s" WHERE id IN %%s""" % self._table
         self._cr.execute(query, [tuple(ids)])
         ids = [r[0] for r in self._cr.fetchall()]
-        existing = self.browse(ids + new_ids)
-        if len(existing) < len(self):
-            # mark missing records in cache with a failed value
-            exc = MissingError(
-                _("Record does not exist or has been deleted.")
-                + '\n\n({} {}, {} {})'.format(_('Records:'), (self - existing).ids[:6], _('User:'), self._uid)
-            )
-            self.env.cache.set_failed(self - existing, self._fields.values(), exc)
```

Comment:
I think that this change requires other ones since that 13.0 changed a lot this part.

We could just do a small change to show the model.

What about only:
```diff
- (self - existing).ids[:6]
+ (self - existing)[:6]
```

Check the difference in the output:
<img width="1070" alt="Screen Shot 2020-05-20 at 17 43 05" src="https://user-images.githubusercontent.com/6644187/82504576-5f82e180-9ac1-11ea-8e05-1d875da510b4.png">

What do you think?
cc @luisg123v

Reply:
I agree.

BTW, isn't this removing the whole message?

Reply:
Sorry, I miss the full patch:

```diff
        query = """SELECT id FROM "%s" WHERE id IN %%s""" % self._table
        self._cr.execute(query, [tuple(ids)])
        ids = [r[0] for r in self._cr.fetchall()]
        existing = self.browse(ids + new_ids)
        if len(existing) < len(self):
            # mark missing records in cache with a failed value
            exc = MissingError(
                _("Record does not exist or has been deleted.")
-                + '\n\n({} {}, {} {})'.format(_('Records:'), (self - existing).ids[:6], _('User:'), self._uid)
+                + '\n\n({} {}, {} {})'.format(_('Records:'), (self - existing).ids, _('User:'), self._uid)
            )
            self.env.cache.set_failed(self - existing, self._fields.values(), exc)
        return existing
```